lib/bemhtml/index.js

```diff
@@ -130,10 +130,10 @@ BEMHTML.prototype.render = function render(context,
     }
 
     if (cls)
-      out += ' ' + cls;
+      out += ' ' + (cls.trim ? utils.attrEscape(cls.trim()) : cls);
   } else {
     if (cls)
-      out += cls;
+      out += cls.trim ? utils.attrEscape(cls.trim()) : cls;
```

Comment:
Em. What else you can pass here?

Why not just to do `cls = utils.attrEscape(cls).trim();` above?